discord_bots/cogs/common.py

```diff
@@ -0,0 +1,559 @@
+                        if game.category_name and category.name == game.category_name
+                    ]
+                    cols = get_table_col(category_games)
+                    table = table2ascii(
+                        header=["Last", "W", "L", "T", "Total", "WR"],
+                        body=cols,
+                        first_col_heading=True,
+                        style=PresetStyle.plain,
+                        alignments=[
+                            Alignment.LEFT,
+                            Alignment.DECIMAL,
+                            Alignment.DECIMAL,
+                            Alignment.DECIMAL,
+                            Alignment.DECIMAL,
+                            Alignment.RIGHT,
+                        ],
+                    )
+                    description += code_block(table)
+                    embed = Embed(title=title, description=description)
+                    if i == (num_pct - 1):
+                        description += f"\n{trueskill_url}"
+                        embed.set_footer(text=footer_text)
+                    embeds.append(embed)
+            if not player_category_trueskills:
+                # no categories defined, display their global trueskill stats
+                description = ""
+                if SHOW_TRUESKILL:
+                    description = f"Rating: {round(player.rated_trueskill_mu - 3 * player.rated_trueskill_sigma, 2)}"
+                    description += (
+                        f"\n{MU_LOWER_UNICODE}: {round(player.rated_trueskill_mu, 1)}"
+                    )
+                    description += f"\n{SIGMA_LOWER_UNICODE}: {round(player.rated_trueskill_sigma, 1)}"
+                else:
+                    description = f"Rating: {trueskill_pct}"
+                cols = get_table_col(fgs)
+                table = table2ascii(
+                    header=["Period", "Wins", "Losses", "Ties", "Total", "Win %"],
+                    body=cols,
+                    first_col_heading=True,
+                    style=PresetStyle.plain,
+                    alignments=[
+                        Alignment.LEFT,
+                        Alignment.DECIMAL,
+                        Alignment.DECIMAL,
+                        Alignment.DECIMAL,
+                        Alignment.DECIMAL,
+                        Alignment.DECIMAL,
+                    ],
+                )
+                description += code_block(table)
+                embed = Embed(
+                    title="Overall Stats",
+                    description=description,
+                )
+                embed.set_footer(text=footer_text)
+                embeds.append(embed)
+            try:
+                await interaction.response.send_message(embeds=embeds, ephemeral=True)
+            except Exception:
+                _log.exception(f"Caught exception trying to send stats message")
+
+    # @app_commands.command(name="status", description="Display queue status")
```

Comment:
Please use a block comment instead haha, I hope you didn't add each # by hand
Eg:
```python
"""
Multiline 
block 
comment
"""
```

Reply:
I just used the `Ctrl+/` shortcut in VS Code to be honest, and that's how it comes out.

Swapped over to docstring instead, which is convenient cause VS Code will let me collapse that now.